khal/settings/khal.spec

```diff
@@ -138,6 +138,10 @@ days = integer(default=2)
 # event in text form, the path to where the event is now saved or nothing?
 print_new = option('event', 'path', 'False', default=False)
 
+# If set to 1, khal will highlight days with events. Options for
+# highlighting are in [highlight_days] section.
+highlight_event_days = integer(default=0)
```

Comment:
could you make this a boolean? (it's what we are using for the other yes no decisions) Or are you planning to make this even more configurable?